src/modules/roomMembers/RoomMembersApi.ts

```diff
@@ -40,14 +55,15 @@ const deleteUser: RequestHandler = async (req: Request, res: Response, next: Nex
 
         const deleted = await service.delete(userId, roomId);
 
-        return res.status(200). json({ deleted });
+        return res.status(200).json({ deleted });
     } catch(e){
         console.error('🔥 error: %o', e);
         return next(e);
     }
 };
 
 export default (app: Router): void => {
+    app.get('/room-members/:id', Auth.authorize([]), list);
```

Comment:
Como vai ficar o endpoint para obter a informação de um room member? E que id é esse? Vê os meus comentários no outro PR se este não for claro

Reply:
O `id` é o `roomId`, mas já alterei isso `:room` [aqui](https://github.com/lopesnuno/chat/pull/20/commits/be93d8415820efa0deaf6477f04cf4a6baaf6373).
Também deixei em comentário, o list é para listar os users de uma sala e o get é para obter informação de um user em especifico.